source/docs/articles/sites/code/using-the-pantheon-workflow.md

```diff
@@ -6,6 +6,8 @@ category:
   - developing
 keywords: pantheon workflow, workflow, dev, test, live, environment, pantheon environments, multidev, how to deploy, deploy, what is a commit, commit, write permission, write to live, why cant i write to live, write access, commits, separate environment, environment, environments
 ---
+<img src="/source/docs/assets/images/dev-test-live-icon.png" alt="Dev Test and Live icon" style="margin-top:0px;margin-left:auto;margin-right:auto;display:block;border:0;" height="100"width="318">
+
 Every Pantheon site comes with three environments: Dev, Test, and Live. Separate Dev, Test, and Live environments allow you to develop and test your site without impacting the live site that's available to the world. Additional development environments are available with [Multidev](/docs/articles/sites/multidev/).
 
 
```

Comment:
@mishabuggy - could you help generate an icon/visual showing "code moves up, content moves down"? @ari-gold suggests something similar to: 
<img width="411" alt="screen shot 2015-07-21 at 4 44 41 pm" src="https://cloud.githubusercontent.com/assets/10119525/8813276/ce9945e2-2fc7-11e5-9701-befc739e5faf.png">